src/Rule/BlankLineAfterFilepathInYamlCodeBlock.php

```diff
@@ -25,8 +25,6 @@ class BlankLineAfterFilepathInYamlCodeBlock extends AbstractRule implements Rule
 {
     public function check(Lines $lines, int $number): ?string
     {
-        $lines = $lines->toIterator();
```

Comment:
That was on my todo list as well 😊 it was just a “Zwischenschritt” in the past 😁